src/merge.rs

```diff
@@ -69,19 +69,19 @@ pub async fn queue(
                     // statuses we actually cared about have all passed, so we "should"
                     // be ok
                     MergeableState::Clean | MergeableState::HasHooks | MergeableState::Unstable => {
-                        let mut merge = prh
+                        let merge = prh
                             .merge(pr_number)
-                            .title(pr.title)
+                            .title(format!("{} (#{})", pr.title, pr_number))
```

Comment:
Does this actually create a hyperlink to the pr?

Reply:
Think GitHub automatically creates a link for it, just as it does in with the commit messages when they reference issues/PRs. 

But good to verify!

Reply:
It does, I checked this commit https://github.com/EmbarkStudios/octobors/commit/9c0e83735980d2e7dcc0338cebbfc75ee5ed44ba